clang/runtime/dpct-rt/include/dpct/dpl_extras/dpcpp_extensions.h

```diff
@@ -566,6 +566,49 @@ class radix_sort {
   uint8_t *_local_memory;
 };
 
+/// Load linear segment items into block format across threads
+/// Helper for Block Load
+template <size_t GROUP_WORK_ITEMS, typename InputT, int ITEMS_PER_THREAD,
+          typename InputIteratorT>
+__dpct_inline__ void load_blocked(size_t linear_tid, InputIteratorT block_itr,
+                                  InputT (&items)[ITEMS_PER_THREAD]) {
+#pragma unroll
+  for (int ITEM = 0; ITEM < ITEMS_PER_THREAD; ITEM++) {
```

Comment:
You should take a step back and consider the naming conventions for variables, template types, template values here.  

It is somewhat a mess already within this file and , but I'd recommend the following to match the compatibility headers in general:
`PascalCase` with for template type parameters
`snake_case`  for local variables
I'm not sure if there is a strong convention already for template values like `GROUP_WORK_ITEMS`.  This file seems to use `ALL_CAPS_WITH_UNDERSCORES`, so I suppose that is OK.  

Whatever we use, lets be consistent, it is confusing to see this local variable `ITEM` in all caps.

Reply:
Bumping this, as I think it still needs addressing for `GROUP_WORK_ITEMS` in `load_striped`.